src/generate_syscalls_perf.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -xeuo pipefail
```

Comment:
Since this is part of the build process, let's avoid any fancy bash settings and do any artifact cleanup as part of `make clean`.

Reply:
FWIW, the reasoning for this is that sometimes it is nice to see the temporary build artifacts when things go wrong.